docs/src/pages/demos/tables/EnhancedTable.js

```diff
@@ -178,9 +191,10 @@ class EnhancedTable extends React.Component {
       order = 'asc';
     }
 
-    const data = this.state.data.sort(
-      (a, b) => (order === 'desc' ? b[orderBy] > a[orderBy] : a[orderBy] > b[orderBy]),
```

Comment:
Yeah, I think that the logic was broken 👍

Reply:
It returned `true` and `false` instead of a negative or positive number.